pkg/kubelet/kubelet.go

```diff
@@ -120,7 +120,7 @@ const (
 
 	// Generic PLEG relies on relisting for discovering container events.
 	// The period directly affects the response time of kubelet.
-	plegRelistPeriod = time.Second * 3
+	plegRelistPeriod = time.Second * 1
```

Comment:
Will do. It's a tradeoff between resource usage and how responsive kubelet will appear.

- We run `docker ps -a` every period to detect the changes, and inspect the changed containers if they exist. This operation could become more expensive as the number of containers grow. Shorter period would cause higher cpu consumption.
- After receiving a new pod, kubelet starts the containers and wait ~1 relist period to detect the changes and report the running status. Increasing the period will cause longer delay.

I'll comment more in the code.